libs/feature/search/src/lib/sort-by/sort-by.component.ts

```diff
@@ -34,6 +39,12 @@ export class SortByComponent {
     private searchService: SearchService
   ) {}
 
+  ngOnInit(): void {
+    if (!this.isQualitySortable) {
```

Comment:
I would prefer that you do the opposite, cause this method won't work if we want a setting for another sort property.

If it's enable, then add the option in the `choices` array.